src/components/Avatar/Avatar.module.css

```diff
@@ -1,10 +1,12 @@
 .Avatar {
   display: flex;
   flex-direction: column;
-  height: 80vh;
-  width: 15%;
+  width: inherit;
   min-width: 260px;
+  height: 80vh;
   align-items: center;
+  position: fixed;
+  top: 10%;
```

Comment:
If we position it using a percentage, on small screen sizes, the `Edit button` won't be at the bottom.